### PR TITLE
Gcp nodes

### PR DESCRIPTION
**Pull request by paigerube14**
This PR will add the node chaos scenarios for GCP clusters

### CODE REVIEW

**Review by mffiedler — CHANGES_REQUESTED** [review]

File: docs/node_scenarios.md [hunk 1]
```diff
@@ -15,7 +15,14 @@ Following node chaos scenarios are supported:
...
 
-**NOTE**: AWS is the only cloud platform supported as of today but we are looking into adding more. Make sure [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) is installed.
+####AWS 
+
+**NOTE**: For clusters with AWS make sure [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) is installed.
+
+####GCP
+**NOTE**: For clusters with GCP make sure [GCP CLI](https://cloud.google.com/sdk/docs/install#linux) is installed.
+
+You'll need to add the following: export GOOGLE_APPLICATION_CREDENTIALS="~/.secrets/serviceaccount.json"
```
[the remark this answers is not in the document]
Comment (paigerube14) [reply docs/node_scenarios.md]: Updated doc, I found a different link to creating a service account. Do you think the link is sufficient or should I add both (the one you included above)?

File: docs/node_scenarios.md [hunk 2]
```diff
@@ -15,7 +15,18 @@ Following node chaos scenarios are supported:
 
 **NOTE**: node_start_scenario, node_stop_scenario, node_stop_start_scenario, node_termination_scenario, node_reboot_scenario and stop_start_kubelet_scenario are supported only on AWS as of now.
```
Comment (mffiedler) [inline docs/node_scenarios.md:16]: Should have caught this before.  I think it is safe to say supported on AWS, GCP and Azure.   In the Azure commit you have AWS and Azure.   I will merge Azure first, but I am guessing we'll have some sort of merge conflict.

Comment (paigerube14) [reply docs/node_scenarios.md]: Updated to say Aws, azure and gcp

File: docs/node_scenarios.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mffiedler) [reply docs/node_scenarios.md]: Yours is better.   Thanks.

**Review by mffiedler — APPROVED** [review]
/LGTM - tested all supported scenarios on GCP and all worked well.

**Pull request merged**